src/System.Security.Cryptography.X509Certificates/src/System.Security.Cryptography.X509Certificates.csproj

```diff
@@ -39,7 +44,12 @@
   <PropertyGroup Condition="'$(TargetGroup)' == 'netcore50'">
     <DefineConstants>$(DefineConstants);NETNATIVE</DefineConstants>
   </PropertyGroup>
-  <ItemGroup>
+  <ItemGroup Condition=" '$(TargetGroup)'=='net461'">
```

Comment:
Can you put the TargetingPackReference at the end of the file so that it doesn't override the System folder in VS.